backend/migrations/20200729222935-kingdom-troops.js

```diff
@@ -25,8 +25,8 @@ exports.up = function (db) {
     hp: { type: 'int', notNull: true },
     attack: { type: 'int', notNull: true },
     defence: { type: 'int', notNull: true },
-    started_at: { type: 'timestamp', notNull: true },
-    finished_at: { type: 'timestamp', notNull: true },
+    started_at: { type: "timestamp", notNull: true, defaultValue: new String('CURRENT_TIMESTAMP') },
+    finished_at: { type: "timestamp", notNull: true, defaultValue: new String('CURRENT_TIMESTAMP') }
```

Comment:
@gulyaasjaanos don't modify existing migrations, create a new one and modify fields, though are you sure that this is a good idea?

Reply:
I'm not sure at all, but migration failed without it.